### PR TITLE
add MIPS and ThirdEye licenses

### PR DESCRIPTION
**Pull request by xsuchy**
Fixes: #2586

### CODE REVIEW

File: src/ThirdEye.xml
Comment (swinslow) [inline src/ThirdEye.xml:4]: Just needs 3.26.0 added for listVersionAdded

File: src/MIPS.xml
Comment (swinslow) [inline src/MIPS.xml:13]: Would add `<copyrightText>` tags around copyright statement

File: src/ThirdEye.xml
Comment (swinslow) [inline src/ThirdEye.xml:13]: Would add `<copyrightText>` tags around copyright statement

File: src/ThirdEye.xml
Comment (swinslow) [inline src/ThirdEye.xml:20]: I'd suggest breaking these up into separate `<p>` sections for each one, since in the original and test text they show up as separate paragraphs.

**Review by swinslow — COMMENTED** [review]
Thank you @xsuchy! I have a few tweaks suggested below which I'll make in a follow-up commit, then will merge if all still passes.

**Comment by swinslow** [comment]
@xsuchy Can you set this PR to allow maintainers to submit commits? If not, I'll just send a follow-up PR, no worries.

**Comment by swinslow** [comment]
Note to self: I'll also update notes to use @jlovejoy's language in https://github.com/spdx/license-list-XML/issues/2586#issuecomment-2539619951

**Comment by xsuchy** [comment]
> Can you set this PR to allow maintainers to submit commits?

This is set (and was from beginning). If you are unable to edit, I will amend the PR in an hour.

I was taking Jilayne comment "no markup" literally. :)

**Comment by xsuchy** [comment]
Updated.

**Comment by swinslow** [comment]
> > Can you set this PR to allow maintainers to submit commits?
> 
> This is set (and was from beginning). If you are unable to edit, I will amend the PR in an hour.
> 
> I was taking Jilayne comment "no markup" literally. :)

Sorry about that! I am sure I did something wrong on trying to push the commit from Git, apologies  :)

Thanks for adding the edits!

**Review by swinslow — APPROVED** [review]

**Pull request merged**
